internal/cloudinfo/providers/amazon/cloudinfo.go

```diff
@@ -617,7 +617,7 @@ func (e *Ec2Infoer) GetServiceProducts(region, service string) ([]types.ProductD
 func (e *Ec2Infoer) GetVersions(service, region string) ([]types.LocationVersion, error) {
 	switch service {
 	case svcEks:
-		return []types.LocationVersion{types.NewLocationVersion(region, []string{"1.19.15", "1.20.7", "1.21.5", "1.22.6"}, "1.21.5")}, nil
+		return []types.LocationVersion{types.NewLocationVersion(region, []string{"1.19.16", "1.20.15", "1.21.13", "1.22.10", "1.23.7"}, "1.22.10")}, nil
```

Comment:
I think we can start dropping old versions like 1.19 and 1.20, 21-23 should be enough IMO.

Reply:
Maybe in another round, if that is okay 🙂